gnat2goto/COMPILING.md

```diff
@@ -0,0 +1,45 @@
+# Build Dependencies
+
+**To build GnatColl GPL 2016 and gnat2goto you must use a gnat GPL 2016 compiler and gprbuild (see the GnatColl user documentation)**
+
+We need the 2016 versions of the GNAT Ada compiler, the gprbuild system and the
+GNATColl utility library to build GNAT2GOTO.
+
+Note: you will *not* be able to build using the debian/ubuntu gnat and gprbuild, it *has* to be the 2016 release from AdaCore.
+
+For **Debian**, install the following debian packages:
+  - build-essential
+
+For **Mac OSX**, you will need the XCode Command Line tools, which require XCode itself and then a 
+```
+xcode-select --install
+```
+to make them available.
+
+Go to [GNAT Community Download page at AdaCore](https://www.adacore.com/download/more),
```

Comment:
This is an action that is common to both OSX and Debian, but the way this documentation is structured makes it look like you only need to do this for Mac OS X.  I'd suggest moving this before the Debian/OSX sections, maybe? Or giving it a heading that makes it clearer that this is needed for both platforms?

Reply:
Actually, it has to be after the Debian/OSX sections, of course. So maybe give this section a "Installing GNAT and GNATCOLL" title?

Reply:
Has heading and separator line now